schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals ( 
+    ID INT PRIMARY KEY NOT NULL, 
+    NAME TEXT NOT NULL, 
```

Comment:
- [ ] The `name` should be of `varchar`, kindly consider implementing this so the name has the proper type